.travis.yml

```diff
@@ -61,13 +62,13 @@ jobs:
       script:
         - unbuffer node build-system/pr-check/validator-tests.js
     - stage: build
-      name: 'Dist, Bundle Size'
+      name: 'Nomodule Dist'
```

Comment:
How prevalent are these names in the JS community? (I just want to make sure that this isn't something we know but others would be confused about)

Reply:
Fair point. Hope about calling the three jobs `Unminified Build`, `Nomodule Build`, and `Module Build`?

Reply:
the JS community use the "module" and "nomodule" terminology pretty consistently. 

Alsoyes i think calling these "build" is more appropriate, will apply recommendation

Reply:
done.